src/prefect/cli/server.py

```diff
@@ -88,6 +96,71 @@ def generate_welcome_blurb(base_url, ui_enabled: bool):
     return blurb
 
 
+def prestart_check():
+    """
+    Check if `PREFECT_API_URL` is set in the current profile. If not, prompt the user to set it.
+    """
+
+    current_profile = load_current_profile()
+    if PREFECT_API_URL not in current_profile.settings:
+        app.console.print(
+            "`PREFECT_API_URL` is not set. You need to set it to communicate with the server.",
+            style="yellow",
+        )
+
+        choice = prompt_select_from_list(
+            app.console,
+            "How would you like to proceed?",
+            [
+                (
+                    "create",
+                    "Create a new profile with `PREFECT_API_URL` set and switch to it",
+                ),
+                (
+                    "set",
+                    f"Set `PREFECT_API_URL` in the current profile: {current_profile.name!r}",
+                ),
+            ],
+        )
+        profiles = load_profiles()
+
+        if choice == "create":
+            while True:
+                profile_name = prompt("Enter a new profile name")
+                if profile_name in profiles:
+                    app.console.print(
+                        f"Profile {profile_name!r} already exists. Please choose a different name.",
+                        style="red",
+                    )
+                else:
+                    api_url = prompt(
+                        "Enter the `PREFECT_API_URL` value",
+                        default="http://127.0.0.1:4200/api",
+                    )
+                    break
+            profiles.add_profile(
+                Profile(name=profile_name, settings={"PREFECT_API_URL": api_url})
+            )
+            profiles.set_active(profile_name)
+            save_profiles(profiles)
+
+            app.console.print(
+                f"Create and switched to new profile {profile_name!r}", style="green"
```

Comment:
```suggestion
                f"Switched to new profile {profile_name!r}", style="green"
```

I think the "new profile" part implies you created it